seedlet-blocks/style-editor.css

```diff
@@ -15,4 +15,9 @@ body {
 	--global--color-primary: var(--wp--preset--color--primary);
 	--global--color-secondary: var(--wp--preset--color--secondary);
 	--global--color-tertiary: var(--wp--preset--color--tertiary);
-}
\ No newline at end of file
+}
+
+.site-content .wp-block[data-align="full"] > [data-block], .site-content .wp-block.alignfull > [data-block] {
+    margin-top: 0;
+    margin-bottom: 0;
```

Comment:
These should be [indented with tabs](https://www.youtube.com/watch?v=SsoOG6ZeyUI)! 🙂

Reply:
Good catch! Thank you.

Reply:
LOL 😂